Orange/base.py

```diff
@@ -664,6 +664,13 @@ def __init__(self, cat_model, cat_features, domain):
         self.cat_model = cat_model
         self.cat_features = cat_features
 
+    def __call__(self, data, ret=Model.Value):
+        if isinstance(data, Table):
+            with data.force_unlocked(data.X):
```

Comment:
Reason?

Reply:
You marked this as resolved; I assume you tried to remove `force_unlocked` and saw what happens -- catboost doesn't declare the array as `const`. I can add a comment, if you think it's necessary.

Reply:
At first, I thought of complaining about the missing comment, but this decorator is only used twice and you commented it so thoroughly that I withdrew any objections.